src/pages/kimjihun/Login/Login.js

```diff
@@ -1,13 +1,84 @@
 import React from 'react';
-
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
 class LoginHun extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userIdValue: '',
+      userPasswordValue: '',
+    };
+  }
+  handleInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
```

Comment:
- 다음과 같이 한칸씩 띄워서 코드끼리 구분 지어주세요!
```suggestion
import { withRouter } from 'react-router-dom';
import './Login.scss';

class LoginHun extends React.Component {
  constructor() {
    super();
    this.state = {
      userIdValue: '',
      userPasswordValue: '',
    };
  }
  
  handleInput = e => {
    this.setState({
      [e.target.name]: e.target.value,
    });
  };
```